src/TestFramework/TestFramework/Assertions/Assert.Contains.cs

```diff
@@ -381,32 +253,23 @@ public static void DoesNotContain<T>(T expected, IEnumerable<T> collection, IEqu
     /// <param name="collection">The collection.</param>
     /// <param name="comparer">An equality comparer to compare values.</param>
     /// <param name="message">The message to display when the assertion fails.</param>
-    public static void DoesNotContain<T>(T expected, IEnumerable<T> collection, IEqualityComparer<T> comparer, string? message)
+    public static void DoesNotContain<T>(T expected, IEnumerable<T> collection, IEqualityComparer<T>? comparer = null, string message = "")
```

Comment:
This makes it not possible to do `Assert.DoesNotContain(t, arrayOfT, "MyMessage")` without using named parameters. Probably not very ideal.

Maybe we should also add `DoesNotContain<T>(T expected, IEnumerable<T> collection, string message = "")`